agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -64,6 +64,8 @@ function collect_sos {
 		quiet="--quiet"
 		block="-o block"
 		processor="-o processor"
+		# ref for plugin collectd: https://github.com/sosreport/sos/pull/866
+		collectd="-o collectd"
```

Comment:
So this got merged into 3.4.  I think we need to test if the minor version is the right version as well.